zebra/zebra_gr.c

```diff
@@ -595,11 +592,7 @@ static int32_t zebra_gr_delete_stale_route(struct client_gr_info *info,
 		 * Reset the current prefix to indicate processing completion
 		 * of the current AFI
 		 */
-		if (info->current_prefix) {
-			XFREE(MTYPE_TMP, info->current_prefix);
-			info->current_prefix = NULL;
-		}
-		continue;
```

Comment:
Was removing continue intentional? It was added as we need to go to next AFI/SAFI.

Reply:
Yes? This loop is effectively
```
for (...) {
...
  continue;
}
```
Making the continue pointless, right? Unless I am misreading?

Reply:
You are right. I misread it and continue is not required here.